thewalrus/_hafnian.py

```diff
@@ -640,7 +640,9 @@ def loop_hafnian(A, D=None, reps=None, glynn=True):
 
 def input_validation(A, rtol=1e-05, atol=1e-08):
     """Checks that the matrix A satisfies the requirements for Hafnian calculation.
+
```

Comment:
This seems to be the same issue as the last time. For some reason the text editor of one of you @brandonpolymtl @JQZ1111 @benjaminlanthier @dleclerc33  is adding empty lines. This need to be corrected!

Reply:
Would suggest to check the wrapping settings of your editors. Also, check if the documentation is rendering correctly.